docs/design/proportional.md

```diff
@@ -1,3 +1,7 @@
+## Deprecated
+
+The `proportion` is deprecated. It is replaced by [capacity](capacity-scheduling.md).
```

Comment:
`proportional` and `proportion` are different plugins: )
And `proportion` is actually not deprecated, `proportion` and `capacity` are just two ways using queue and cover different use cases.

Reply:
The hierarchical queue function may not be compatible with proportion.